dotnet/src/SemanticKernel.UnitTests/Planning/PlanSerializationTests.cs

```diff
@@ -435,15 +435,16 @@ public async Task CanStepAndSerializeAndDeserializePlanWithStepsAndContextAsync(
             .Returns(() => Task.FromResult(returnContext));
         mockFunction.Setup(x => x.Describe()).Returns(new FunctionView()
```

Comment:
```suggestion
        mockFunction.Setup(x => x.Describe()).Returns(new FunctionView
```

Reply:
could you suggest that to the original author?